inttests/fs_test.go

```diff
@@ -18,7 +18,6 @@ import (
 	"testing"
 
 	"github.com/dell/goscaleio"
-	// log "github.com/sirupsen/logrus"
```

Comment:
can we remove commented out line?

Reply:
Hey @JacobGros , yes as part of this PR I removed the commented out references to `sirupsen/logrus`.